### PR TITLE
new(tests): EOF - EIP-3540: test all opcodes in valid code section

### PR DESCRIPTION
**Pull request by winsvega**
## 🗒️ Description
Put every instruction with valid stack / data in between a valid eof code section

## ✅ Checklist
- [x] All: Set appropriate labels for the changes.
- [x] All: Considered squashing commits to improve commit history.
- [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
- [x] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
- [x] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
- [x] Tests: Included the type and version of evm t8n tool used to locally execute test cases:  e.g., ref with commit hash or geth 1.13.1-stable-3f40e65.
- [x] Tests: Ran `mkdocs serve` locally and verified the auto-generated docs for new tests in the [Test Case Reference](https://ethereum.github.io/execution-spec-tests/main/tests/) are correctly formatted.

### CODE REVIEW

File: tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+def make_opcode_valid_bytes(opcode: Opcode) -> Opcode | bytes:
+    """
+    Construct a valid stack and bytes for the opcode
+    """
+    if opcode._name_ in eof_opcodes and opcode != Op.RETF:
+        if opcode == Op.CALLF:
+            return opcode[1]
+        return opcode[0]
+    return opcode + make_data_portion(opcode)
```
Comment (marioevz) [inline tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py:189]:
```suggestion
def make_opcode_valid_bytes(opcode: Opcode) -> Opcode:
    """
    Construct a valid stack and bytes for the opcode
    """
    if opcode.data_portion_length == 0 and opcode.data_portion_formatter is None:
        return opcode
    if opcode == Op.CALLF:
        return opcode[1]
    return opcode[0]
```
Just to make this forward compatible with #631

**Review by marioevz — APPROVED** [review]
Fantastic, thanks!

**Comment by winsvega** [comment]
Hi, I think all tests can benefit from UndefinedOpcodes enum.
Also I think Unrechable instruction exception if for another test case. here we just check that if opcode is valid and in eof then no exception happens. (the only exception here is retf) 
I applied your commit also

**Comment by marioevz** [comment]
> is this related to my code or a global issue?

Global issue, I'll help rebase because I just merged a big PR :+1:

**Review by marioevz — APPROVED** [review]
Thanks!

**Pull request merged**